settings/local_settings.py

```diff
@@ -45,8 +44,6 @@
     }
     print("[BACKEND] Using sqlite3 database", flush=True)
 
-ALLOWED_HOSTS: list[str | None] = ["localhost", "127.0.0.1"]
```

Comment:
Could this be left with | None as described in https://github.com/TreyWW/MyFinances/pull/324#discussion_r1563955230